metrics.md

```diff
@@ -294,6 +294,7 @@ $$ O_i = \begin{cases}
 The CRPS reduces to the mean absolute error (MAE) if the forecast is deterministic.
 
 ## References
-[Espinar] Bella Espinar, Lourdes Ramírez, Anja Drews, Hans Georg Beyer, Luis F. Zarzalejo, Jesús Polo, Luis Martín,
+{: .anchor}
+[<span id="ref-espinar">Espinar</span>] Bella Espinar, Lourdes Ramírez, Anja Drews, Hans Georg Beyer, Luis F. Zarzalejo, Jesús Polo, Luis Martín,
```

Comment:
looks like maybe this could be simplified to `<a name="ref-espinar"></a> Bella Espinar, Lourdes Ramírez,...`

https://stackoverflow.com/questions/5319754/cross-reference-named-anchor-in-markdown/7335259#7335259